### PR TITLE
Implement account creation modal - Closes #487

### LINKED ISSUE #487: Migrate account creation modal
- [x] Passphrase creation utility
- [x] Unit test coverage for the utilities
- [x] Create account modal
- [x] Unit test coverage for modal
- [ ] e2e test coverage for modal and triggering button

### PR DESCRIPTION
**Pull request by reyraa**
Closes #487

### CODE REVIEW

File: src/components/passphrase/passphraseConfirmator.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    this.props.updateAnswer(value === this.state.missing);
+  }
+
+  render() {
+    return (
+      <div className={`${grid.row} ${grid['start-xs']}`}>
+        <div className={grid['col-xs-12']}>
+          <p>
+            <span>{this.state.passphraseParts[0]}</span>
+            <span className="missing">-----</span>
```
Comment (reyraa) [inline src/components/passphrase/passphraseConfirmator.js]: Done

File: src/components/passphrase/passphrase.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      answer: '',
+    };
+  }
+
+  changeHandler(name, value) {
+    this.setState({ [name]: value });
+  }
+
+  render() {
+    const templates = {};
```
Comment (reyraa) [inline src/components/passphrase/passphrase.js]: I've just moved configs there, not templates. templates are big enough to stuff whichever file we put them in.

File: src/components/passphrase/passphrase.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+          </div>
+        </section>
+        <section className={`${grid.row} ${grid['between-xs']}`}>
+          <Button label={this.state.steps[this.state.currentStep].cancelButton.title}
+            className='cancel-button'
+            onClick={this.state.steps[this.state.currentStep].cancelButton.onClick.bind(this)} />
+
+          <Button label={this.state.steps[this.state.currentStep].confirmButton.title}
+            primary={true} raised={true}
+            disabled={(this.state.currentStep === 'generate' && !this.state.passphrase) ||
```
Comment (reyraa) [inline src/components/passphrase/passphrase.js]: To put them there, I have to call `steps` function in `componentDidMount` to access `state` and that means I'm rendering the whole component once more. although I get your point, but I believe it doesn't worth another time rendering. but I still can do it if you insist.

File: src/components/login/loginFormComponent.js [hunk 4]
```diff
@@ -137,7 +138,14 @@ class LoginFormComponent extends React.Component {
         />
         <footer className={ `${grid.row} ${grid['center-xs']}` }>
           <div className={grid['col-xs-12']}>
-            <Button label='NEW ACCOUNT' flat primary />
+            <Button label='NEW ACCOUNT' flat primary
```
Comment (reyraa) [inline src/components/login/loginFormComponent.js]: Done

File: src/components/passphrase/steps.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+      title: 'next',
+      onClick: () => { context.setState({ currentStep: 'generate' }); },
+    },
+  },
+  generate: {
+    cancelButton: {
+      title: 'cancel',
+      onClick: () => { context.props.closeDialog(); },
+    },
+    confirmButton: {
```
Comment (slaweet) [inline src/components/passphrase/steps.js]: the disabled 'Yes! It\'s safe' button looks weird in the `generate` step. There used to be no buttons in this step in Angular. That should be not too difficult to do in here, too.

Comment (reyraa) [inline src/components/passphrase/steps.js]: Done

File: src/components/passphrase/steps.js [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+      title: 'Yes! It\'s safe',
+      onClick: () => {},
+    },
+  },
+  show: {
```
Comment (slaweet) [inline src/components/passphrase/steps.js]: I just realized that the `show` step is missing the title "Save your passphrase in a safe place!".  Without the title the "yes it's safe" button doesn't make sense

File: src/components/passphrase/steps.js (hunk 5 above)
Comment (slaweet) [inline src/components/passphrase/steps.js]: The last step had also its own title - "Enter the missing word to continue". But there it isn't that big problem without it.

Comment (reyraa) [inline src/components/passphrase/steps.js]: Yes I knew about that. This is an issue with Dialog component. There are also some other improvements required on that component. I'll create a ticket for it

**Pull request merged**